src/pms/cli.py

```diff
@@ -10,20 +10,24 @@
     import importlib_metadata as metadata
 
 from loguru import logger
-from typer import Argument, Context, Exit, Option, Typer, echo
+from typer import Argument, Context, Exit, Option, Typer, echo, run
 
 from pms.core import MessageReader, SensorReader, Supported, exit_on_fail
 
-main = Typer(help="Data acquisition and logging for Air Quality Sensors with UART interface")
-
+app = Typer(help="Data acquisition and logging for Air Quality Sensors with UART interface")
 
 """
 Extra cli commands from plugins
 
 additional Typer commands are loaded from plugins (entry points) advertized as `"pypms.extras"`
 """
 for ep in metadata.entry_points(group="pypms.extras"):
-    main.command(name=ep.name)(ep.load())
+    app.command(name=ep.name)(ep.load())
+
+
+def main():  # pragma: no cover
+    logger.enable("pms")
```

Comment:
can this not be done on the CLI callback function?

Reply:
Good question. I did try this originally but it doesn't work for testing: some of the other tests execute the callback, which overrides the package default (that logging is disabled).

Since the purpose of the test I added is to check for regression in the package default, I needed to put this line somewhere where no other test would cause it to run.

I admit it's not a great change or a great test, but since this is the second PR to try and make logging work better for library users, I wanted to put something in to protect the behaviour.